src/muse/outputs/mca.py

```diff
@@ -770,14 +770,13 @@ def sector_fuel_costs(
             data_agent["category"] = a.category
             data_agent["sector"] = getattr(sector, "name", "unnamed")
             data_agent["year"] = output_year
-            if len(data_agent) > 0 and len(data_agent.technology.values) > 0:
-                data_sector.append(data_agent.groupby("technology").fillna(0))
+            data_agent = multiindex_to_coords(data_agent, "timeslice").to_dataframe(
+                "fuel_consumption_costs"
+            )
+            if not data_agent.empty:
+                data_sector.append(data_agent)
```

Comment:
This looks ok, but it is doing something different than before. I assume this is correct, but I'm not sure why it is.

Reply:
ah, ok, you are combining it with what it was being done afterwards. OK, looks fine.